content/applications/services/helpdesk/advanced/close_tickets.rst

```diff
@@ -1,54 +1,46 @@
 ======================================
-Allow Customers to Close their Tickets
+Allow customers to close their tickets
 ======================================
 
 Allowing customers to close their tickets gives them autonomy and minimize misunderstandings about
-when an issue is considered solved, or not. It makes communication and actions more efficient.
+when an issue is considered solved or not.
 
 Configure the feature
 =====================
 
-To configure the feature go to :menuselection:`Helpdesk --> Settings --> Helpdesk Teams --> Edit`
-and enable *Ticket closing*.
+To configure the feature, go to :menuselection:`Helpdesk --> Configuration --> Helpdesk Teams`,
+click on the team, and click :guilabel:`Edit` and enable :guilabel:`Ticket closing`.
 
-.. image:: close_tickets/closetickets1.png
-   :align: center
-   :alt: Ticket closing in Odoo Helpdesk
+.. image:: close_tickets/close-ticket-settings.png
+   :alt: Ticket closing feature in Odoo Helpdesk.
 
-In order to designate to which stage the ticket migrates to once it is closed, go to
-:menuselection:`Helpdesk --> Overview --> Tickets`.
+To designate which stage the ticket migrates to once it is closed, navigate to the ticket pipeline
+by going to :menuselection:`Helpdesk --> Overview` and clicking :guilabel:`Tickets` on the team's
+card.
 
-.. image:: close_tickets/closetickets2.png
-   :align: center
-   :alt: Ticket closing in Odoo Helpdesk
+There are two options: create a new Kanban stage or work with an existing one. For both scenarios,
+click the gear icon next to the stage name, select :guilabel:`Edit Stage`, and enable
+:guilabel:`Closing Stage`.
```

Comment:
This is a good start.
What's missing here are guilabels and a little bit more context as to what the reader should be looking for e.g. field types should be specified in the text. As well for v14-15, **Save** needs to be clicked to save the changes.

```suggestion
click the :guilabel:`Settings (gear)` icon next to the stage name, select :guilabel:`Edit Stage`,
and enable :guilabel:`Closing Stage`. After checking the field box, click :guilabel:`Save` to
finish.
```